ext/openssl/openssl.c

```diff
@@ -1064,16 +1064,26 @@ static inline int php_openssl_config_check_syntax(const char * section_label, co
 }
 /* }}} */
 
+static char *conf_get_string_quiet(
```

Comment:
Could you pls rename it to something like `php_openssl_conf_get_string` so it's consistent with other functions and clear that it's not OpenSSL functions.